tls/s2n_certificate_verify.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "crypto/s2n_hash.h"
+#include "error/s2n_errno.h"
+#include "stuffer/s2n_stuffer.h"
+#include "tls/s2n_tls13_handshake.h"
+#include "tls/s2n_certificate_verify.h"
+#include "tls/s2n_connection.h"
+#include "utils/s2n_safety.h"
+
+#include <stdint.h>
+
+/**
+  * Specified in https://tools.ietf.org/html/rfc8446#section-4.4.3
+  *
+  * Servers MUST send this message when authenticating via a certificate.  
+  * Clients MUST send this message whenever authenticating via a certificate. 
+  * When sent, this message MUST appear immediately after the Certificate 
+  * message and immediately prior to the Finished message.
+ **/
+
+/* 64 'space' characters (0x20) */
+const uint8_t S2N_CERT_VERIFY_PREFIX[] = {0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
```

Comment:
Maybe use [memset](http://man7.org/linux/man-pages/man3/memset.3.html) instead?

Reply:
My hesitation with memset is then I would lose the ability to declare it const